imap_processing/lo/l0/compression_tables.py

```diff
@@ -0,0 +1,853 @@
+"""Compression tables for decompompressing Lo L0 data."""
+tof_case_table = {
```

Comment:
Could this just be a function call instead?
`int(bitstring, 2)`

Reply:
oh wow lol yeah...I didn't notice that. I'll remove this table and convert the binary to decimal